cmd/core-command/Attribution.txt

```diff
@@ -154,6 +154,9 @@ https://github.com/golang/net/blob/master/LICENSE
 gopkg.in/yaml.v2 (Apache 2.0) https://github.com/go-yaml/yaml/
 https://github.com/go-yaml/yaml/blob/v2.2.2/LICENSE
```

Comment:
V2 is used by our code. V3 is indirect dependency used by testify. We really should not need to list V3 here since it is only used in the test code, but the script doesn't know that.

Reply:
ok, agree; but it seems to bring up one issue that our testing system for attribution file should skip that based on that reasoning then. maybe worth some discussion in future WG meeting. @lenny-intel